qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Filters for vacancies
+
+Background:
+Given User is on https://career.quantori.com/ru/positions
+And vacancies available with Security and Compliance Competence
+And vanacies available with Linux Techology
+
+Scenario: Drop-down of Competence and Technologies
+When User clicks the fields Competence or Technologies
+Then User sees drop-down menu with available Competencies or Technologies
```

Comment:
I would recommend to use Scenario outline for this case instead of using "or". It will help to check both of the fields and make tests more specific.